js/entries/foundation-plugins.d.ts

```diff
@@ -0,0 +1,21 @@
+export { Abide } from '../foundation.abide';
```

Comment:
> js/entries/foundation-plugins.d.ts

Shouldn't the new folder be ts

Reply:
for compatibility I think we should not rename everything

Reply:
Yeah same like how kotlin does with java
Gotcha ... Agree!